mcp/streamable.go

```diff
@@ -594,12 +597,38 @@ type StreamableClientTransport struct {
 	opts StreamableClientTransportOptions
 }
 
+// StreamableClientReconnectionOptions defines parameters for client reconnection attempts.
+type StreamableClientReconnectionOptions struct {
+	// InitialDelay is the base delay for the first reconnection attempt
+	InitialDelay time.Duration // default: 1 second
```

Comment:
There's no need to doc the defaults since they're visible in the exported default value for this struct.